librz/analysis/p/analysis_8051.c

```diff
@@ -837,54 +1058,54 @@ static bool set_reg_profile(RzAnalysis *analysis) {
 		"gpr	c	.1	.103	0\n"
 		"gpr	sp	.8	13	0\n"
 		"gpr	pc	.16	15	0\n"
-// ---------------------------------------------------
-// 8051 memory emulation control registers
-// These registers map 8051 memory classes to r2's
-// linear address space. Registers contain base addr
-// in r2 memory space representing the memory class.
-// Offsets are initialized based on asm.cpu, but can
-// be updated with ar command.
-//
-// _code
-//		program memory (CODE)
-// _idata
-//		internal data memory (IDATA, IRAM)
-// _sfr
-//		special function registers (SFR)
-// _xdata
-//		external data memory (XDATA, XRAM)
-// _pdata
-//		page accessed by movx @ri op (PDATA, XREG)
-//		r2 addr = (_pdata & 0xff) << 8 + x_data
-//		if 0xffffffnn, addr = ([SFRnn] << 8) + _xdata (TODO)
+		// ---------------------------------------------------
+		// 8051 memory emulation control registers
+		// These registers map 8051 memory classes to r2's
+		// linear address space. Registers contain base addr
+		// in r2 memory space representing the memory class.
+		// Offsets are initialized based on asm.cpu, but can
+		// be updated with ar command.
+		//
+		// _code
+		//		program memory (CODE)
+		// _idata
+		//		internal data memory (IDATA, IRAM)
+		// _sfr
+		//		special function registers (SFR)
+		// _xdata
+		//		external data memory (XDATA, XRAM)
+		// _pdata
+		//		page accessed by movx @ri op (PDATA, XREG)
+		//		r2 addr = (_pdata & 0xff) << 8 + x_data
+		//		if 0xffffffnn, addr = ([SFRnn] << 8) + _xdata (TODO)
```

Comment:
unnecessary indent

Reply:
I think both ways are fine. I wasn't able to persuade `clang-format` to not do this.